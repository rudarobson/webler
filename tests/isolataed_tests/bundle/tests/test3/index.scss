@import '_index1.scss';
@import '_index2.scss';
