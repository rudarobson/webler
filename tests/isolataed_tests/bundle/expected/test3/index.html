<!DOCTYPE html>
<html>

<head>
  <link type="text/css" rel="stylesheet" href="index.css" />
</head>

<body>
</body>

</html>
